packages/xstate-lit/src/UseMachine.ts

```diff
@@ -0,0 +1,85 @@
+import { ReactiveController, ReactiveControllerHost } from 'lit';
+import {
+  Actor,
+  AnyStateMachine,
+  ActorOptions,
+  createActor,
+  EventFrom,
+  SnapshotFrom,
+  Subscription
+} from 'xstate';
+
+export class UseMachine<TMachine extends AnyStateMachine>
+  implements ReactiveController
+{
+  private host: ReactiveControllerHost;
```

Comment:
I'd recommend prefixing private fields with `_` so that they're more obviously private even to plain JavaScript users.

Even better if you can, I'd use standard private fields. They have very good [browser support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties#javascript.classes.private_class_fields) these days, and even better tool support.

Reply:
As with my previous comment, I followed the style used in xstate here. I reviewed some controllers in Lit, such as:

- [context-consumer](https://github.com/lit/lit/blob/main/packages/context/src/lib/controllers/context-consumer.ts#L40)
- [mutation-controller](https://github.com/lit/lit/blob/main/packages/labs/observers/src/mutation-controller.ts#L63)

And decided to maintain that style.

Reply:
Oops... those don't actually follow our code style, so I'll fix them. Thanks for pointing them out!

One reason why we prefix all private fields is so that they can be renamed in a minifier like Terser with a simple rule.